.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -1,7 +1,7 @@
 name: Bug report
 description: File a bug report
 title: "Bug title"
-labels: [bug]
+labels: [bug] [needs-triage]
```

I think this should be something like this:
```suggestion
labels: ["bug", "needs-triage"]
```